gradle.properties

```diff
@@ -19,3 +19,18 @@ android.useAndroidX=true
 android.enableJetifier=true
 # Kotlin code style for this project: "official" or "obsolete":
 kotlin.code.style=official
+POM_LICENCE_URL=http://opensource.org/licenses/MIT
+POM_DEVELOPER_ID=Rudderstack
+POM_NAME=Rudder-Integration-CleverTap-Android
+POM_DESCRIPTION=RudderStack\'s Native SDK Integration support for CleverTap.
+GROUP=com.rudderstack.android.integration
+POM_LICENCE_DIST=repo
+VERSION_NAME=1.0.1
+POM_LICENCE_NAME=The MIT License (MIT)
+POM_DEVELOPER_NAME=Rudderstack, Inc.
+VERSION_CODE=3
```

Comment:
1.0.1 should correspond to version code 2, is there any reason it's 3?

Reply:
I forgot to update that value. Now I've pushed the changes.